src/main/kotlin/lotto/Application.kt

```diff
@@ -1,5 +1,19 @@
 package lotto
 
 fun main() {
-    // TODO: 프로그램 구현
+    val inputView = InputView()
+    val inputValidator = InputValidator()
+    val inputController = InputController(inputView = inputView, inputValidator = inputValidator)
+    val outputView = OutputView()
+    val lottoGenerator = LottoGenerator()
+    val lottoWinningResultCalculator = LottoWinningResultCalculator()
+    val profitRateCalculator = ProfitRateCalculator()
+    val lottoController = LottoController(
+        inputController = inputController,
+        lottoWinningResultCalculator = lottoWinningResultCalculator,
+        profitRateCalculator = profitRateCalculator,
+        outputView = outputView,
+        lottoGenerator = lottoGenerator
+    )
+    lottoController.run()
```

Comment:
개인적인 취향이지만 이 방식은 어떤거같나용??
```suggestion
    val inputController = InputController(
        inputView = InputView(),
        inputValidator = InputValidator(),
    )
    val lottoController = LottoController(
        inputController = inputController,
        lottoWinningResultCalculator = LottoWinningResultCalculator(),
        profitRateCalculator = ProfitRateCalculator(),
        outputView = OutputView(),
        lottoGenerator = LottoGenerator()
    )
    lottoController.run()
```